README.md

```diff
@@ -6,23 +6,25 @@ A simple command line application to test JDBC connection to Oracle Database.
 
 Clone this repository and then run:
 
-```
+```sh
 mvn clean package
 ```
-Or download the JAR file from [release](https://github.com/aimtiaz11/jdbc-tester/releases) page. 
 
-Execute the JAR file with the following 3 parameters with schema name, password and JDBC connection details:
+Or download the JAR file from [release](https://github.com/aimtiaz11/jdbc-tester/releases) page.
 
+Execute the JAR file with the following 2 parameters JDBC connection details and optional query, authentication credentials must be set as environment variables:
+
+```sh
+env ORACLEDB_USER=myuser ORACLEDB_PASS=mypass java -jar target/jdbc-tester-1.0.jar jdbc:oracle:thin:@//<host>:<port>/<SID> [optional_query]
```

Comment:
This `env ORACLEDB_USER=myuser ORACLEDB_PASS=mypass`
will still get logged in .bash_history, so doesn't add any value in terms of security to compared to how its originally done.

One better option would be to read username & password using java.util.Scanner. 
https://alvinalexander.com/java/edu/pj/pj010005/

I was planning to implement it but have lost access to an Oracle DB instance to finish & test this out.

Reply:
using **env** is no the only option, to avoid a history register of the password an external file with both variables set can be sourced, assuming that a batch option usage of this tool is available.

Expecting a password input would defeat a batch option, think about it. I think it could ask for a password input only if is not set in a env variable.

Reply:
(credentials file)
```
ORACLEDB_USER=myuser
ORACLEDB_PASS=mypass
```

```
source credentials
java -jar target/jdbc-tester-1.0.jar jdbc:oracle:thin:@//<host>:<port>/<SID> [optional_query]
```

Reply:
If you want a even more secure option, such option could be **envconsul** , providing Hashicorp's Vault secrets to environment variables, such as:

```
envconsul -config myconfig.hcl java -jar target/jdbc-tester-1.0.jar jdbc:oracle:thin:@//<host>:<port>/<SID> [optional_query]
```

My point here is that environment variables is the most versatile choice for input values, with that you can leave the secure handling of such values to the user of the jar ;)

Reply:
Agree about not prompting for credential. 
But i don’t think we should force the user to use specific env variable (ie. 
ORACLEDB_USER, ORACLEDB_PASS)

I’ve put a comment in README.md about using the ‘read’ command to read them in into any variable Name  of their choosing and use them as best security practice. 

What variable they choose and how they set it would get outside scope of this app. That I thinks the best way to securely use this without modifying any input params.